lib/cli/src/frameworks/common/0.Introduction.stories.mdx

```diff
@@ -0,0 +1,194 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import Code from './assets/code-brackets.svg';
+import Colors from './assets/colors.svg';
+import Comments from './assets/comments.svg';
+import Direction from './assets/direction.svg';
+import Flow from './assets/flow.svg';
+import Plugin from './assets/plugin.svg';
+import Repo from './assets/repo.svg';
+import StackAlt from './assets/stackalt.svg';
+
+<Meta title="Example/Introduction" />
+
+<style>{`
```

Comment:
Do you think we should maybe import this style too?

Reply:
I think we should if it's easy. I tried to import a css file in mdx but it doesn't work. Is that supposed to be supported ?

If it's not, maybe we should log a ticket and see if it gets community support ? 

Either it's wanted or not, I think we can ship this and improve later.